.github/workflows/fmu-tools.yml

```diff
@@ -19,17 +19,30 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
-        pandas-version: ["0.21.0", "1.*"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        # 1.1.3 for Python 3.8 in RMS up to 14.2
+        # 1.5.3 for Komodo
+        pandas-version: ["1.1.3", "1.5.3", "2.*"]
```

Comment:
I have pandas `2.*` in the test matrix so it will test it against all Python versions, but `2.0.2` could be specified for Python 3.11 in addition to this (2.* will take the latest version)